sql/src/test/java/io/crate/action/sql/SessionTest.java

```diff
@@ -215,4 +217,63 @@ public void testExtractTypesFromInsertWithOnDuplicateKey() throws Exception {
 
         assertThat(parameterTypes, is(new DataType[]{DataTypes.STRING, DataTypes.STRING}));
     }
+
+    @Test
+    public void testDeallocateAll() {
```

Comment:
`testDeallocateAllClearsAllPortalsAndPreparedStatements` (though I'm not sure if clearing the portals is correct, can we figure that out somehow?)

Reply:
Seems we only need to clear prep statements and not portals:
https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/deallocate.go
https://github.com/postgres/postgres/blob/153a49bb331005bf70b1e76e69fe28f1c417cc91/src/interfaces/ecpg/ecpglib/prepare.c#L253